context.go

```diff
@@ -313,9 +313,17 @@ func checkRequiredFlags(flags []Flag, context *Context) requiredFlagsErr {
 	var missingFlags []string
 	for _, f := range flags {
 		if rf, ok := f.(RequiredFlag); ok && rf.IsRequired() {
-			key := strings.Split(f.GetName(), ",")[0]
-			if !context.IsSet(key) {
-				missingFlags = append(missingFlags, key)
+			key := strings.Split(f.GetName(), ",")
+			if len(key) > 1 {
+				// has short name
+				if !context.IsSet(strings.TrimSpace(key[0])) && !context.IsSet(strings.TrimSpace(key[1])) {
```

Comment:
What if we have more than one additional short name, like:
```go
			Name:  "long, lo, l"
```
`l` would not covered in that case, right?

Reply:
Is that something we support? I have not seen multiple short names for a single flag even in the documentation. Anyway, if this is supported, I guess iteration would the way to go.